### PR TITLE
New Gun Variants Part 1: Old NT Ballistics (and some minor fixes)

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
Always wished that gun parts had more stuff to make since the start, and i missed the old design of the MK58 pistol, so why not kill two birds with one stone. Old NT guns with Old Old NT Color Scheme
05-Jul-22 15-39-45

## Why It's Good For The Game
More gun modularity and pretty sprites good.

## Changelog
:cl:
add: Military Variants of the MK58 and Regulator with slightly better recoil that can be made with rubber grips
fix: Edited excel AK name to fit their naming scheme (From AKMS to Kalashnikov)
fix: Fixed Miller frame making a Mateba
tweak: Shotgun descriptions now state Capacity + 1 in the chamber
imageadd: added new alternate MK58 and Gladstone Sprites
/:cl:

### CODE REVIEW

**Comment by maikilangiolo** [comment]
Ok now stop making new gun variants with different stats, and implement base gun profiles with alternative sprites

**Comment by ValoTheValo** [comment]
> Ok now stop making new gun variants with different stats, and implement base gun profiles with alternative sprites

it has like a minute amount better recoil like .1 because rubber grips are slightly harder to find than plastic grips

**Review by SirRichardFrancis — COMMENTED** [review]
Kalashnikov-variants-is-absolutely-evil 2: Electric Boogaloo

**Review by Firefox13 — COMMENTED** [review]
shotgun tube size should be generated automatiicaly in a different line but who cares really

**Review by TheShown911 — COMMENTED** [review]
i LOVE guns

File: code/modules/projectiles/guns/projectile/shotgun/bull.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/item/gun/projectile/shotgun/bull
 	name = "FS SG \"Bull\""
 	desc = "A \"Frozen Star\" double-barreled pump-action shotgun. Marvel of engineering, this gun is often used by Ironhammer tactical units. \
-			Due to shorter than usual barrels, damage are somewhat lower and recoil kicks slightly harder, but possibility to fire two barrels at once overshadows all bad design flaws."
+			Due to shorter than usual barrels, damage are somewhat lower and recoil kicks slightly harder, but possibility to fire two barrels at once overshadows all bad design flaws. it holds 7+2 shells."
```
Comment (hyperioo) [inline code/modules/projectiles/guns/projectile/shotgun/bull.dm:4]:
```suggestion
			Due to shorter than usual barrels, damage are somewhat lower and recoil kicks slightly harder, but possibility to fire two barrels at once overshadows all bad design flaws. Can hold up to 7+2 shells."
```

File: code/modules/projectiles/guns/projectile/shotgun/pump.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/shotgun/pump
 	name = "FS SG \"Kammerer\""
-	desc = "When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds."
+	desc = "When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds. it holds 4+1 shells in its tube"
```
Comment (hyperioo) [inline code/modules/projectiles/guns/projectile/shotgun/pump.dm:3]:
```suggestion
	desc = "When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds. Can hold up to 4+1 shells in its tube"
```

**Pull request merged**